src/ACadSharp/ACadSharp.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net5.0;net6.0;net7.0;net48;netstandard2.1</TargetFrameworks>
+		<TargetFrameworks>net5.0;net6.0;net7.0;net48;netstandard2.0</TargetFrameworks>
```

Comment:
I cannot see the reason of this downgrade.

Could you justify why should be implemented for 2.0 instead of 2.1?
Can you add 2.0 without removing the 2.1?